schedule/security/selinux_jeos.yaml

```diff
@@ -0,0 +1,42 @@
+name: selinux_jeos
+description: >
+    This is a SElinux test for JeOS Tumbleweed
+schedule:
+    - installation/bootloader_uefi
+    - jeos/firstrun
+    - jeos/record_machine_id
+    - console/force_scheduled_tasks
+    - jeos/grub2_gfxmode
+    - jeos/diskusage
+    - jeos/build_key
+    - console/prjconf_excluded_rpms
+    - console/journal_check
+    - microos/libzypp_config
+    - update/zypper_clear_repos
+    - console/zypper_ar
+    - console/zypper_ref
+    - console/consoletest_setup
+    - security/selinux/selinux_setup
+    - security/selinux/sestatus
+    - '{{sle_os_tests}}'
+    - security/selinux/semanage_fcontext
+    - security/selinux/semanage_boolean
+    - security/selinux/fixfiles
+    - security/selinux/print_se_context
+    - security/selinux/audit2allow
+    - security/selinux/semodule
+    - security/selinux/setsebool
+    - security/selinux/restorecon
+    - security/selinux/chcon
+    - security/selinux/chcat
+    - security/selinux/set_get_enforce
+    - security/selinux/selinuxexeccon
+conditional_schedule:
+    sle_os_tests:
+        DISTRI:
+            sle:
+                - security/selinux/selinux_smoke
+                - security/selinux/enforcing_mode_setup
+            opensuse:
+                - security/selinux/selinux_smoke
+                - security/selinux/enforcing_mode_setup
```

Comment:
Same as Ondrej mentioned, we dont really need the condition as the result is the same

Reply:
Removed the conditional part.